src/endpoints/swagger/swagger_controller.ts

```diff
@@ -0,0 +1,17 @@
+import {HttpRequestWithIdentity} from '@essential-projects/http_contracts';
+
+import * as path from 'path';
+
+import {HttpController} from '@process-engine/consumer_api_contracts';
+
+import {Response} from 'express';
+
+export class SwaggerController implements HttpController.ISwaggerHttpController {
+
+  private httpCodeSuccessfulResponse: number = 200;
+
+  public async getSwaggerJson(request: HttpRequestWithIdentity, response: Response): Promise<void> {
+    response.status(this.httpCodeSuccessfulResponse).sendFile(path.join(__dirname, '..', '..', '..', '..', 'swagger.json'));
```

Comment:
Besser `path.resolve`, damit der Pfad aufgelöst wird.

```suggestion
    response.status(this.httpCodeSuccessfulResponse).sendFile(path.resolve(__dirname, '..', '..', '..', '..', 'swagger.json'));
```